docs/admin-guide/admin-configuration.md

```diff
@@ -413,6 +414,54 @@ The following table lists the configurable attributes for enhanced search.
 | `cve`            | Extends enhanced search to allow searching of Common Vulnerabilities and Exposures (CVE).                                                  |
 | `updateInterval` | Sets the interval at which the searchable database of CVE items is refreshed.                                                              |
 
+<a name="userprefs_config"></a>
+
+## Setting user preferences
+
+The user preferences extension provides an API endpoint for adding configurable user preferences for a repository. This custom extension, not a part of the OCI distribution, is accessible only by authenticated users of the registry. Unauthenticated users are denied access.
+
+To allow users to set preferences, add the `userprefs` attribute under `extensions` in the configuration file.
```

Comment:
Currently `userprefs` is enabled by default when the search extension is enabled and there isn't a configuration under extensions to enable disable `userprefs`.